R/package.R

```diff
@@ -6,58 +6,6 @@
 #'   \code{\link{kernels}} and \code{\link{gp}}
 #'
 #' @docType package
-#' @importFrom gpflowr gpflow
 #' @importFrom greta .internals
 #'
 NULL
-
```

Comment:
The package is erroring for me when running the examples. E.g.:

```r
library(greta.gp)
k1 <- rbf(lengthscales = c(0.1, 0.2), variance = 0.6)
```
> ```
> Error in lapply(parameters, as.greta_array) : 
>   object 'as.greta_array' not found 
> ```

I think that's because you don't get the unexported function `as.greta_array()` from greta. You can do that with:
```r
as.greta_array <- greta::.internals$greta_arrays$as.greta_array
```
and here is a good place to put it. you may need to do that for other internal functions too

Reply:
have added as.greta_array and op to package.R. Not sure if op is needed.